@xen-orchestra/backups/_cleanVm.js

```diff
@@ -37,60 +37,36 @@ const computeVhdsSize = (handler, vhdPaths) =>
 // and all the others will deleted
 async function mergeVhdChain(chain, { handler, onLog, remove, merge }) {
   assert(chain.length >= 2)
-
-  let child = chain[0]
-  const parent = chain[chain.length - 1]
-  const children = chain.slice(0, -1).reverse()
-
-  chain
-    .slice(1)
-    .reverse()
-    .forEach(parent => {
-      onLog(`the parent ${parent} of the child ${child} is unused`)
-    })
+  // chain is parent -> child -> grand child > ..
```

Comment:
```suggestion
  // chain is [ unusedParent, unusedChild..., usedChild ]
```

Reply:
```
// chain is [ parent, child, grandChild ]
// we want to copy the used blocks of childs in parent 
// then rename the parent as the last child
// and delete all the other vhd , since they won't be used anymore
```